### PR TITLE
Finish Router II

### PR DESCRIPTION
**Pull request by iVictoris**
Complete tasks requested, fix bug that Movie component was causing due to initial render where `stars` is undefined, added ternary check `{stars ? <do map...> : null }`

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,388 @@
+{
```
Comment (JossWritesCode) [inline package-lock.json:1]: You have both a package-lock.json and a yarn.lock file. That means you ran both npm and yarn. If you do that by accident, make sure you delete either the package-lock.json to get rid of npm or the yarn.lock file to get rid of yarn.

File: client/src/App.js [hunk 2]
```diff
@@ -1,6 +1,10 @@
 import React, { useState } from 'react';
 
+import {Route} from 'react-router-dom';
+
 import SavedList from './Movies/SavedList';
```
Comment (JossWritesCode) [inline client/src/App.js:5]: Have you thought about trying to add the SavedList feature? It's a fun stretch problem. You can message me for hints if you need any :)

File: client/src/Movies/Movie.js [hunk 3]
```diff
@@ -1,25 +1,23 @@
...
+import axios from "axios";
 
-const Movie = (props) => {
+const Movie = props => {
   const [movie, setMovie] = useState({});
- 
+
   useEffect(() => {
-    const id = 1;
+    const id = props.match.params.id;
```
Comment (JossWritesCode) [inline client/src/Movies/Movie.js:8]: Looks great. Pop Quiz: Where is "match" coming from?

**Review by JossWritesCode — COMMENTED** [review]
Your code looks and works great. You show a good understanding of react router :D

File: client/src/Movies/Movie.js (hunk 3 above)
Comment (iVictoris) [reply client/src/Movies/Movie.js:8]: `BrowserRouter` or as `Router` (by convention) component that wraps around `<App />`

**Pull request merged**